pkg/testworkflows/utils.go

```diff
@@ -108,3 +108,12 @@ func GetClusterID(ctx context.Context, configMap configRepo.Repository) string {
 	}
 	return clusterID
 }
+
+// CountMapBytes returns the total bytes of the map
+func CountMapBytes(m map[string]string) int {
```

Comment:
It's only approximate, as the data will be stored in MongoDB as a BSON structure. As an example:
* `"♥♥♥♥♥♥♥♥♥♥"`
   * `len(str)` -> 30
   * BSON size -> 115 bytes
* `"XXXXXXXXXX"`
   * `len(str)` -> 10
   * BSON size -> 95 bytes

Considering that, I think that the best would be to decrease the limit above to - some still reasonable, but lower - 1-3 MB, to ensure that we won't hit limits due to conversion.